src/video-player.js

```diff
@@ -322,8 +322,10 @@ class VideoPlayer extends Utils.mixin(Eventable) {
 
   _initChapters() {
     this.videojs.on(PLAYER_EVENT.CLD_SOURCE_CHANGED, (e, { source }) => {
-      if (!isEmpty(source._chapters)) {
-        this.videojs.chapters(source._chapters);
+      if (!isEmpty(source._chapters) && this.videojs.chapters) {
+        isFunction(this.videojs.chapters)
+          ? this.videojs.chapters(source._chapters)
+          : this.videojs.chapters.src(source._chapters);
```

Comment:
It looks a bit complex to me, is this changing behaviour from videojs or from our code?

Reply:
It's just that the plugin needs to be called differently on init and on source update. Same for our other custom plugins. We can probably change that.